### PR TITLE
Matt user page

### PR DESCRIPTION
**Pull request by matthewcrooke1**
Clicking your name redirects to a profile page with stats. If you click somebody else's username it will go to your page, still working on that though.

### CODE REVIEW

**Comment by ZachDFK** [comment]
Might have to check with the others but it works for me :+1:

File: app/assets/stylesheets/profile.css.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+html, head, body { margin: 0; padding: 0; width: 100%; height: 100% }
+
+
+body {
+  display: flex;
+  flex-flow: column;
+}
```
Comment (kevincox) [inline app/assets/stylesheets/profile.css.scss:7]: Why is this here? Styling global elements from one page is probably not what you want.

File: app/models/user.rb [hunk 2]
```diff
@@ -13,13 +13,26 @@ class User < ActiveRecord::Base
...
+  acts_as_voter
+  has_secure_password
+
+  def trustscore
+      up_score = 0.0
+      vote_score = 0.0
+      self.posts.each do |post|
+        up_score += post.get_upvotes.size
+        vote_score += post.votes_for.size
+      end
```
Comment (kevincox) [inline app/models/user.rb:29]: This should really be a single query rather then pulling in every post.

File: app/views/sessions/profile.html.haml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+%h1 Welcome #{current_user.to_s}, to your profile page!
```
Comment (kevincox) [inline app/views/sessions/profile.html.haml:1]: The `to_s` is unnecessary.
